contracts/test/unit/PCVGuardian.sol

```diff
@@ -0,0 +1,491 @@
+        pcvGuardian.withdrawERC20RatioToSafeAddress(
+            address(pcvDeposit1),
+            address(pcvDeposit2),
+            address(token),
+            1,
+            false,
+            false
+        );
+        vm.stopPrank();
+
+        vm.startPrank(addresses.pcvSafeMoverAddress);
+        pcvGuardian.withdrawToSafeAddress(address(pcvDeposit1), address(pcvDeposit2), 1, false, false);
+        pcvGuardian.withdrawETHToSafeAddress(address(pcvDeposit1), payable(address(pcvDeposit2)), 1, false, false);
+        pcvGuardian.withdrawERC20ToSafeAddress(
+            address(pcvDeposit1),
+            address(pcvDeposit2),
+            address(token),
+            1,
+            false,
+            false
+        );
+        pcvGuardian.withdrawRatioToSafeAddress(address(pcvDeposit1), address(pcvDeposit2), 1, false, false);
+        pcvGuardian.withdrawETHRatioToSafeAddress(address(pcvDeposit1), payable(address(pcvDeposit2)), 1, false, false);
+        pcvGuardian.withdrawERC20RatioToSafeAddress(
+            address(pcvDeposit1),
+            address(pcvDeposit2),
+            address(token),
+            1,
+            false,
+            false
+        );
+        vm.stopPrank();
+
+        vm.startPrank(addresses.guardianAddress);
+        pcvGuardian.withdrawToSafeAddress(address(pcvDeposit1), address(pcvDeposit2), 1, false, false);
+        pcvGuardian.withdrawETHToSafeAddress(address(pcvDeposit1), payable(address(pcvDeposit2)), 1, false, false);
+        pcvGuardian.withdrawERC20ToSafeAddress(
+            address(pcvDeposit1),
+            address(pcvDeposit2),
+            address(token),
+            1,
+            false,
+            false
+        );
+        pcvGuardian.withdrawRatioToSafeAddress(address(pcvDeposit1), address(pcvDeposit2), 1, false, false);
+        pcvGuardian.withdrawETHRatioToSafeAddress(address(pcvDeposit1), payable(address(pcvDeposit2)), 1, false, false);
+        pcvGuardian.withdrawERC20RatioToSafeAddress(
+            address(pcvDeposit1),
+            address(pcvDeposit2),
+            address(token),
+            1,
+            false,
+            false
+        );
+        vm.stopPrank();
+
+        // move back all tokens & eth
+        vm.startPrank(addresses.governorAddress);
+        pcvGuardian.withdrawETHRatioToSafeAddress(
+            address(pcvDeposit2),
+            payable(address(pcvDeposit1)),
+            10000,
+            false,
+            false
+        );
+        pcvGuardian.withdrawERC20RatioToSafeAddress(
+            address(pcvDeposit2),
+            address(pcvDeposit1),
+            address(token),
+            10000,
+            false,
+            false
+        );
+        pcvDeposit1.deposit();
+        pcvDeposit2.deposit();
+        vm.stopPrank();
+    }
+
+    // check state changes & getters after set & unset
+    function testStateConsistency1() public {
```

Comment:
Would definitely rename this test name. What state consistency are we checking?

Reply:
I would also split this unit test out into several. Each unit test should test as close to just one function as possible imo